packages/lit-auth-client/src/lib/providers/EthWalletProvider.ts

```diff
@@ -90,9 +90,17 @@ export default class EthWalletProvider extends BaseProvider {
         address: address,
       };
     } else {
+      if (!this.litNodeClient.latestBlockhash) {
+        throwError({
+          message: 'Eth Blockhash is undefined.',
```

Comment:
can you make this  more descriptive of why the block hash is undefined? since this is implicitly loaded from `handshake` the end user doesn't have a lot of options.

Reply:
Added a message to reconnect